README.md

```diff
@@ -116,6 +116,18 @@ optional flags:
     	Version prints the release version of validator
 ```
 
+### Environment Variables
+
+The config-file-validator supports setting options via environment variables. If both command-line flags and environment variables are set, the command-line flags will take precedence. The supported environment variables are as follows:
+
+- `CFV_DEPTH`: Equivalent to the `-depth` flag
+- `CFV_EXCLUDE_DIRS`: Equivalent to the `-exclude-dirs` flag
+- `CFV_EXCLUDE_FILE_TYPES`: Equivalent to the `-exclude-file-types` flag
+- `CFV_OUTPUT`: Equivalent to the `-output` flag
+- `CFV_REPORTER`: Equivalent to the `-reporter` flag
+- `CFV_GROUPBY`: Equivalent to the `-groupby` flag
+- `CFV_QUIET`: Equivalent to the `-quiet` flag
```

Comment:
Would prefer these in a markdown table - easier to read/maintain

Reply:
👍 Just changed it